src/components/App.jsx

```diff
@@ -41,21 +41,21 @@ const App = () => {
   }
 
   return (
-    <Router>
+    <Router future={{ v7_startTransition: true, v7_relativeSplatPath: true }}>
       <Fragment>
         <Header />
         <div className="content">
           <Sidebar />
           <main className="main">
-            <Switch>
-              <Route exact path="/" component={NewSnippet} />
-              <Route exact path="/about" component={About} />
-              <Route exact path="/recent" component={RecentSnippets} />
-              <Route exact path="/sign-in" component={SignIn} />
-              // Switch renders the first matched route, so we need to keep this
-              // one last as the regexp below matches the paths of other pages
-              <Route exact path="/:id([a-zA-Z0-9]+)" component={Snippet} />
-            </Switch>
+            <Routes>
+              <Route path="/" element={<NewSnippet />} />
+              <Route path="/about" element={<About />} />
+              <Route path="/recent" element={<RecentSnippets />} />
+              <Route path="/sign-in" element={<SignIn />} />
+              {/* Routes renders the first matched route, so we need to keep this
+                  one last as the regexp below matches the paths of other pages */}
+              <Route path="/:id" element={<Snippet />} />
```

Comment:
`([a-zA-Z0-9]+)` should have been preserved?

Reply:
Probably. But I don't think it's important. Worst case we will stop showing 404 on something that doesn't look like a legit snippet ID.

Reply:
Probably. But I don't think it's important. Worst case we will stop showing 404 on something that doesn't look like a legit snippet ID.